main/mysql/main.go

```diff
@@ -2,8 +2,8 @@ package main
 
 import (
 	"github.com/wal-g/wal-g/cmd/mysql"
-	"github.com/wal-g/wal-g/internal"
-	"github.com/wal-g/wal-g/main"
+	interconfig "github.com/wal-g/wal-g/internal/config"
+	config "github.com/wal-g/wal-g/main"
```

Comment:
You shouldn't want to rename the main package to anything.

Reply:
goreturns did this for me. Fixed.
By the way, directory github.com/wal-g/wal-g/main indeed contains a package named "config".